src/index.js

```diff
@@ -12,6 +12,12 @@ import LineStream from './line-stream';
 import ParseStream from './parse-stream';
 import Parser from './parser';
 
+export {
```

Comment:
seems like some of the newer babel transforms are requiring it, not entirely sure why, let me look into it